myapp/app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Admin::UsersController < ApplicationController
+  before_action :find_user, only: [:update, :show, :destroy]
+
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:notice] = "User created successfully."
```

Comment:
I thought it would be better to use I18n for the message like this.

Reply:
Done. Thanks for your advice